flow/designs/asap7/riscv32i/config.mk

```diff
@@ -14,6 +14,22 @@ export MAX_UNGROUP_SIZE ?= 1000
 
 export VERILOG_FILES = $(sort $(wildcard ./designs/src/$(DESIGN_NICKNAME)/*.v))
 export SDC_FILE      = ./designs/$(PLATFORM)/$(DESIGN_NICKNAME)/constraint.sdc
+
+# To mock an SRAM means to take the behavioral Verilog and create a
+# scaled down versions while still using all the input/outputs. This can
+# be useful when no fake sram is available from the PDK or the existing
+# fake srams are not a great match for the SRAMs used in the design.
+#
+# Typically this is done by reducing the number of rows, such as from 128
+# to 4 in this case. The unused address bits are xor'ed together the
+# unused address bits so that all input/outputs of the module are still in
+# use.
+#
+# This gives us a mock .lef and .lib file.
+#
+# To try out mock sram, uncomment the "export BLOCKS" line and
+# comment out "export ADDITIONAL_LEFS" and "export ADDITIONAL_LIBS" lines.
+# export BLOCKS=fakeram7_256x32
```

Comment:
With no CI I wonder if this will continue to work.  It might be better to convert the test case to use this if this is important.

Reply:
Fixed. Force pushed to make it easier to review as I've changed the structure of the PR.